### DISCUSSION TITLE
Camelopardalis Constellation: Stellarium vs IAU

### DISCUSSION THREAD

**Discussion by jrobparsons** [Stellarium/stellarium#2126] [General]
Stellarium displays Camelopardalis different from what I see on the IAU chart.

Here is the IAU chart: https://www.iau.org/public/themes/constellations/#cam

Attached is a screenshot of what I see in Stellarium (Windows 10 desktop version).

![stellarium-002](https://user-images.githubusercontent.com/2822864/147369326-9d345a84-fcc1-45ae-901c-8052d1f58b36.png)

Anyone know why the difference?

**Comment by alex-w** [comment]
The lines of constellations is not defined by IAU and no exist "correct" collection of lines. If you want to see lines of constellations in Stellarium as on IAU.org website, then please select "Western (Sky & Telescope)" sky culture.

**Reply by jrobparsons** [reply]
Thanks!
